pypeit/core/extract.py

```diff
@@ -1504,12 +1506,9 @@ def objfind(image, thismask, slit_left, slit_righ, inmask=None, fwhm=3.0, maxdev
             sobjs.add_sobj(thisobj)
 
     nobj = len(sobjs)
-    # If there are no regular aps and no hand aps, just return
-    #if nobj == 0:
-    #    return (None, skymask, objmask)
-
-    ## Okay now loop over all the regular aps and exclude any which within the fwhm of the hand_extract_APERTURES
-    if nobj_reg > 0 and hand_extract_dict is not None:
+    # Okay now loop over all the regular aps and exclude any which within the fwhm of the hand_extract_APERTURES
+    # We're going to assume that if interactive updates were made that the user has manually assigned the traces.
```

Comment:
Can we remove this comment about interactive updates, since it is a bit confusing given that the interactive part is removed now.

Reply:
Good catch - yes I've removed line 1510